AssemblerLib/LocalDataInitializer.h

```diff
@@ -240,18 +242,19 @@ class LocalDataInitializer
 
     /// Sets the provided data_ptr to the newly created local assembler data.
     void operator()(
-            const MeshLib::Element& e,
+            std::size_t const element_id,
```

Comment:
Oh, that index. Yes they can be different. Please don't call it element_id. It is the mesh_item_id which can also refer to node ids (or edge, face ids, when implemented).